cockatrice/src/tab_deck_editor.cpp

```diff
@@ -988,6 +992,17 @@ void TabDeckEditor::actRemoveCard()
     if (!currentIndex.isValid() || deckModel->hasChildren(currentIndex))
         return;
     deckModel->removeRow(currentIndex.row(), currentIndex.parent());
+
+    // Disable saving if the deck is empty
+    DeckLoader *const deck = deckModel->getDeckList();
+    QString decklistUrlString;
+    if (deck) {
+        decklistUrlString = deck->exportDeckToDecklist();
```

Comment:
I know this is only run when a card is removed, but it looks to me like a lot of wasted cpu cycles.
Why not use `deck->isEmpty()`?